name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesChart.java

```diff
@@ -1246,59 +1255,50 @@ private void addDividendMarkerLines(ChartInterval chartInterval)
             if (chartConfig.contains(ChartDetails.SHOW_DATA_LABELS))
             {
                 customPaintListeners.add(event -> {
+                    FontMetrics fontMetrics = event.gc.getFontMetrics();
+
+                    StringBuilder labelBuilder = new StringBuilder();
```

Comment:
The use of the StringBuilder does not make sense here. It will be only filled by exactly one string (the one returned by getDividendLabel.